src/pages/Markets/components/LpCard.tsx

```diff
@@ -31,9 +31,13 @@ export const LpCard: React.FC<LpCardProps> = ({ assetId, apy, volume24H, onClick
     return bnOrZero(volume24H).isPositive() ? <Amount.Fiat value={volume24H} /> : <CText>N/A</CText>
   }, [volume24H])
 
+  const green = useColorModeValue('green.500', 'green.200')
   const apyValue = useMemo(() => {
+    const valuePercent = bnOrZero(apy)
+    if (valuePercent.gt(1000)) return <Text color={green}>1000%+</Text>
```

Comment:
We should probably add a tooltip here so the user can get the actual number, whatever that insane number may be.

I also saw support for making this `10,000`, as some legit opportunities have 5 digit APYs for a time.

Reply:
https://jam.dev/c/54dd7f1b-61ae-460d-b79b-7684883a3a0e

- tackled 10,000% in 2c914d83c8c385a6373c4c2b72d45ec45a54d110 - assuming @shapeshift/product is fine with that
<img width="1396" alt="image" src="https://github.com/user-attachments/assets/6ca3f0c2-dd80-4d88-a081-2940729e6a63" />
- tackled the hover tooltip in e1ad186dc5 (yep, that absurd APY is indeed absurd but correct! Can be double-checked by monkey-patching `isDisabled` to false and checking against other APYs